.env.template

```diff
@@ -12,3 +12,5 @@ DRIVE_GATEWAY_PASSWORD=gatewaypass
 REDIS_HOST=payments-cache
 REDIS_PASSWORD=
 COMEBACK_COUPON_CODE=
+DRIVE_NEW_GATEWAY_URL=http://drive-server-wip:3004/api
+DRIVE_NEW_GATEWAY_SECRET=
```

Comment:
Isn't there a value for this secret in the development environment? If not, you can suggest one and propose a PR on drive-server-wip also, so we keep this environment stable enough to run smoothly on local